src/Microsoft.Identity.Web.TokenCache/Distributed/MsalDistributedTokenCacheAdapter.Logger.cs

```diff
@@ -78,20 +78,18 @@ private static class Logger
             /// <param name="operation">Cache operation (Read, Write, etc...).</param>
             /// <param name="cacheKey">MSAL.NET cache key.</param>
             /// <param name="cacheSize">Cache size in bytes, or 0 if empty.</param>
-            /// <param name="ex">Exception.</param>
             public static void MemoryCacheRead(
                 ILogger logger,
                 string cacheType,
                 string operation,
                 string cacheKey,
-                int cacheSize,
-                Exception? ex) => s_l1CacheRead(
+                int cacheSize) => s_l1CacheRead(
                     logger,
                     cacheType,
                     operation,
                     cacheKey,
                     cacheSize,
-                    ex);
+                    default!);
```

Comment:
what is it?

Reply:
It's a default replacement for the exception when an exception is not used in the log message. https://learn.microsoft.com/en-us/dotnet/core/extensions/high-performance-logging